<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>937</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="load_tracks_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>load_tracks_button</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previous_track_button">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>previous_track_button</string>
    </property>
   </widget>
   <widget class="QPushButton" name="play_pause_button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>play_pause_button</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="next_trak_button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>250</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>next_trak_button</string>
    </property>
   </widget>
   <widget class="QTableView" name="playlist_tabel_view">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>1051</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="current_track_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="track_progress_bar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>1091</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSlider" name="volume_slider">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="value">
     <number>60</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="mute_button">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>mute</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
